blocks/init/src/Blocks/manifest.json

```
@@ -60,27 +60,27 @@
 			{
 				"name": "Ocean",
 				"slug": "ocean",
-				"color": "#44AEDE"
+				"color": "#155D7E"
 			},
 			{
 				"name": "Green",
 				"slug": "green",
-				"color": "#29A3A3"
+				"color": "#175959"
 			},
 			{
 				"name": "Red",
 				"slug": "red",
-				"color": "#E84E4E"
+				"color": "#AF1818"
 			},
 			{
 				"name": "Yellow",
 				"slug": "yellow",
-				"color": "#F99922"
+				"color": "#77460D"
```

```suggestion
				"color": "#FFEA00"
```
Needs to have black text for WCAG AAA.